com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -139,8 +139,9 @@ public Matrix4x4 GetGPUProjectionMatrix(int viewIndex = 0)
         public float renderScale;
         internal ImageScalingMode imageScalingMode;
         internal ImageUpscalingFilter upscalingFilter;
-        public bool clearDepth;
+        public HDRColorBufferFormat hdrColorBufferFormat;
         public CameraType cameraType;
+        public bool clearDepth;
```

Comment:
Needs public API docs?

Reply:
I just moved the bools together as minor cleanup.